### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Juliocho990**
### Summary
- Login form was created
- When you submit, you are redirected to the book list.
### Screenshot
login

### Trello card
https://trello.com/c/madJCpcM/32-login

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  <div class="container">
+    <div class="login__wrapper">
+      <div class="login__card">
+        <div class="login__logo"></div>
+        <form id="loginForm" action="./list.html">
+          <label class="login__label" for="email">Email</label>
+          <input class="login__input" type="email" id="email" required>
+          <label class="login__label" for="password">Password</label>
+          <input class="login__input" type="password" id="password" password required>
+          <button class="login__btn" id="btnSubmit" login>Login</button>
```
Comment (eledwinn) [inline login.html]: If button id is not used, please remove.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        </form>
+        <hr class="divider">
+        <button class="login__btn" sign-up>Sign Up</button>
+      </div>
+    </div>
+  </div>
+</body>
+</html>
```
Comment (eledwinn) [inline login.html]: Empty line here 😄

File: login.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      <div class="login__card">
+        <div class="login__logo"></div>
+        <form id="loginForm" action="./list.html">
+          <label class="login__label" for="email">Email</label>
+          <input class="login__input" type="email" id="email" required>
+          <label class="login__label" for="password">Password</label>
+          <input class="login__input" type="password" id="password" password required>
+          <button class="login__btn" id="btnSubmit" login>Login</button>
+        </form>
+        <hr class="divider">
```
Comment (eledwinn) [inline login.html]: We like to see these dividers as decorative elements. You can even have a class that add this style with a pseudo element.

File: login.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        <button class="login__btn" sign-up>Sign Up</button>
```
Comment (eledwinn) [inline login.html]: Add `type` attr.

File: login.html (hunk 1 above)
Comment (eledwinn) [inline login.html]: Add `submit` type.

Comment (eledwinn) [inline login.html]: Why the `login` attribute? 🤔

File: login.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body class="body--white">
+  <div class="container">
+    <div class="login__wrapper">
+      <div class="login__card">
+        <div class="login__logo"></div>
+        <form id="loginForm" action="./list.html">
+          <label class="login__label" for="email">Email</label>
```
Comment (eledwinn) [inline login.html]: it's better if you use more generic classes for forms. Example: `form__label`, `form__input`.

File: scss/login.scss [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  height: 32px;
+  justify-content: center;
+  width: 100%;
+  
+  &[login] {
+    background-color: $las-palmas;
+    color: white;
+  }
+
+  &[sign-up] {
```
Comment (eledwinn) [inline scss/login.scss]: I prefer `.primary` and `.secondary` class modifiers. What do you think?

File: scss/login.scss [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    border: 2px solid $las-palmas;
+    color: $las-palmas;
+
```
Comment (eledwinn) [inline scss/login.scss:75]: Remove empty line

File: scss/login.scss [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+  }
+}
+
+.divider {
+  margin-bottom: 16px;
+}
+
+.login__btn {
+  margin-bottom: 24px;
+}
```
Comment (eledwinn) [inline scss/login.scss]: Add empty line here

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: login.html (hunk 1 above)
Comment (Juliocho990) [reply login.html]: ok

File: login.html (hunk 3 above)
Comment (Juliocho990) [reply login.html]: great!

File: login.html (hunk 4 above)
Comment (Juliocho990) [reply login.html]: ok

File: login.html (hunk 1 above)
Comment (Juliocho990) [reply login.html]: ok

Comment (Juliocho990) [reply login.html]: This was removed and instead the type attribute

File: scss/login.scss (hunk 6 above)
Comment (Juliocho990) [reply scss/login.scss]: I have implemented these class modifiers. I researched and may be more helpful in this case.

File: scss/login.scss (hunk 7 above)
Comment (Juliocho990) [reply scss/login.scss:75]: ok

File: scss/login.scss (hunk 8 above)
Comment (Juliocho990) [reply scss/login.scss]: ok

**Review by CrisVega08 — APPROVED** [review]

File: login.html (hunk 2 above)
Comment (Juliocho990) [reply login.html]: ok

**Review by eledwinn — APPROVED** [review]

**Pull request merged**